routes/templates/favourites.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * All routes for favourites are defined here
+ * Since this file is loaded in server.js into api/users,
+ *   these routes are mounted onto /users
+ * See: https://expressjs.com/en/guide/using-middleware.html#middleware.router
+ */
+
+const express = require("express");
+const router = express.Router();
+
+
+
+module.exports = (db) => {
```

Comment:
This is where we can use our EJS templateVars to pass in the data we get back from the resource layer into the front end EJS html